org.adempiere.base/src/org/compiere/model/PO.java

```diff
@@ -190,6 +190,20 @@ public PO (Properties ctx, ResultSet rs, String trxName)
 	 *  @param rs optional - load from current result set position (no navigation, not closed)
 	 */
 	public PO (Properties ctx, int ID, String trxName, ResultSet rs)
+	{
+		this(ctx, ID, trxName, rs, null);
+	}
+
+	/**
+	 * Create & Load existing Persistent Object.
+	 * 
+	 * @param ctx     context
+	 * @param ID      the ID if 0, the record defaults are applied - ignored if re exists
+	 * @param trxName transaction name
+	 * @param rs      optional - load from current result set position (no navigation, not closed)
+	 * @param uuID    optional - load from uuID
+	 */
+	public PO(Properties ctx, int ID, String trxName, ResultSet rs, String uuID)
```

Comment:
this constructor should be removed as well

Reply:
Done